packages/exampleReactNative/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "exampleReactNative",
+  "version": "0.0.1",
+  "private": true,
+  "workspaces": {
+    "nohoist": [
+      "react-native",
+      "react-native/**",
+      "react-native-randombytes",
+      "react-native-randombytes/**",
+      "@react-native-community/**",
+      "react-native-svg",
+      "react-native-svg/**"
+    ]
+  },
+  "scripts": {
+    "copy-workspace-packages": "yarn copy-identicon && yarn copy-ui-shared",
+    "copy-identicon": "mkdir -p node_modules/@polkadot/reactnative-identicon && cp -r ../reactnative-identicon/build/* ./node_modules/@polkadot/reactnative-identicon",
+    "copy-ui-shared": "mkdir -p node_modules/@polkadot/ui-shared && cp -r ../ui-shared/build/* ./node_modules/@polkadot/ui-shared",
+    "android": "react-native run-android --no-packager",
+    "ios": "react-native run-ios --no-packager",
+    "start": "NODE_OPTIONS=--max_old_space_size=8192 react-native start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@react-native-community/async-storage": "^1.6.2",
```

Comment:
We need the `@polkadot/{ui-keyring,ui-settings,reactnative-identicon}` deps in here as well (as they are used)

Reply:
MB, wasn't sure since the example-react doesn't list the deps in its package.json

Reply:
Erk, ok, that is a bug :( Will update. :)